app/src/androidTest/java/chat/rocket/android/authentication/login/ui/LoginFragmentTest.kt

```diff
@@ -0,0 +1,87 @@
+package chat.rocket.android.authentication.login.ui
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.IdlingRegistry
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.filters.LargeTest
+import androidx.test.rule.ActivityTestRule
+import chat.rocket.android.R
+import chat.rocket.android.analytics.event.ScreenViewEvent
+import chat.rocket.android.authentication.matchers.withHint
+import chat.rocket.android.authentication.ui.AuthenticationActivity
+import chat.rocket.android.util.espressoIdlingResource.EspressoIdlingResource
+import chat.rocket.android.util.extensions.addFragmentBackStack
+import org.hamcrest.Matchers.not
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+@LargeTest
+class LoginFragmentTest {
+
+    private val serverUrl = "https://open.rocket.chat"
+    private val USERNAME: String = "user121"
+    private val PASSWORD: String = "123456"
```

Comment:
you had to create this user on open.rocket.chat for these tests to work?  If so, can you add that information to the README please?  Any organization that forks RC should know how to run the tests against their own server if needed.  Thanks.

Reply:
@ear-dev Indeed a great suggestion, we have to make the user before running the tests in case of other organization for tests to pass. I have made the user in case of RC. I will mention the info of this type in a proper README. Thanks